lib/utils.bash

```diff
@@ -42,14 +42,34 @@ get_platform() {
 	fi
 }
 
+get_architecture() {
+	arch=$(uname -m)
+	case $arch in
+	aarch64 | arm64)
+		echo "aarch64"
```

Comment:
We could do this:

```suggestion
	aarch64 | arm64)

		if [[ "$(uname -s)" == "Darwin" ]]; then
			# TODO: remove once vector supports darwin ARM; see the following:
			# https://github.com/vectordotdev/vector/issues/10193#issuecomment-1533865422
			echo >&2 "Architecture not supported" && exit 1
		fi
		echo "aarch64"
```

But this would require editing and fixing later when the release becomes available for Darwin ARM ?